persistence/src/main/java/edu/unc/lib/dl/services/OperationsMessageSender.java

```diff
@@ -314,6 +314,31 @@ public String sendEditTypeOperation(String userid, Collection<PID> pids, Resourc
         return getMessageId(msg);
     }
 
+    /**
+     * Sends Update MODS operation message.
+     *
+     * @param userid id of user who triggered the operation
+     * @param pids objects whose MODS changed
+     * @return id of operation message
+     */
+    public String sendUpdateDescriptionOperation(String userid, Collection<PID> pids) {
+        Element contentEl = createAtomEntry(userid, pids.iterator().next(),
+                CDRActions.UPDATE_MODS);
+
+        Element updateEl = new Element("updateMods", CDR_MESSAGE_NS);
```

Comment:
I know we're not doing this elsewhere, but it seems like we could use the CDRAction enum for giving the name to this element, since the action name has to match the element name

Reply:
sure, why not